INSTALL.md

```diff
@@ -121,12 +129,17 @@ Navigate to the cloned directory and ensure your conda environment is active:
 cd qiita
 source activate qiita
 ```
-
+Before we install Qiita, make sure you have a few other tools installed:
+```bash
+sudo apt install gcc
```

Comment:
@aliu104 That's right, ubuntu doesn't install a CC by default. I would rephrase and ask the user to type 'cc' first. If the response is 'program not found' or similar, then install either gcc or clang, depending on their preference. Gcc specifically isn't a requirement - we just need a C++ compiler. This makes the documentation appropriate for organizations using distributions other than Ubuntu, where CC might already be installed.